lib/checks_helper.rb

```diff
@@ -102,4 +102,126 @@ def self.create_check(params, scan_id = nil)
     end
     check
   end
+
+def self.create_check_with_id(params)
```

Comment:
As we've discussed, perhaps would be better to merge this method and the original `create_check` one.
The behaviour included in this method is compatible in the old one and I see like a feature therefore, I would try to keep one single `create_check` method compatible with both scenarios.
WDYT?